### ISSUE TITLE
Maintainers Meeting

### ISSUE THREAD

**Issue by sstevens2** [carpentries-incubator/docker-introduction#50]
Hi all! @aturner-epcc , @ChristinaLK, @jcohen02, @dme26

I think we should meet to discuss goals for the lesson.  With some polishing, we could get this out of beta and it would be helpful to talk about it as a group.

I know we are a difficult bunch to schedule and the holidays are coming up so I'd like to use this issue as a placeholder to get us discussing when we can meet.

I propose we plan to meet in Feb or March. Reply below with...
- Are there any weeks that are not possible for you during that time?
- What UTC time range could we possibly meet in for you.

My answers:
- No particular blocked dates yet but March might be better than Feb as I'm hosting a research Bazaar.
- My UTC time range is 13 UTC - 3 UTC.

**Comment by jcohen02** [comment]
My answers:

- Mondays 13-19 UTC not great for most of Feb/March otherwise March generally looking more practical but late Feb possible.
- Aside from the above, my UTC time range is 7:30 UTC - 21 UTC.

**Comment by ChristinaLK** [comment]
mid Feb through march is fine for me. 
I'm usually free on Mon/Wed afternoon (central time), Tuesday/Friday during the day (10-3ish)
UTC 13 - 3 UTC (like Sarah)

**Comment by aturner-epcc** [comment]
* 8-15 Feb is school half term here so I will be off with the kids
* 22-26 Feb I am involved with a funding panel
* 1-6 Mar I am involved with a funding panel
* Generally UTC 0830-1500,1800-2100

**Comment by dme26** [comment]
Good plan, thanks Sarah!

- First week of February isn’t ideal, but is likely still possible.
- Generally 18 UTC – 13 UTC.

**Comment by sstevens2** [comment]
Ok suggesting March 16th at 18 UTC.  Let me know how that works for you.

**Comment by ChristinaLK** [comment]
Time + date (for convenience) would be: https://www.timeanddate.com/worldclock/fixedtime.html?msg=Docker+Maintainers&iso=20210316T18&p1=%3A&ah=1

**Comment by aturner-epcc** [comment]
Great, it is in my diary. Out of curiosity, why do people say "UTC" rather than "GMT"?

**Comment by dme26** [comment]
Works for me and is in my diary. Thanks!

(Also, for UTC vs GMT: https://currentmillis.com … not that I've directly answered the question…)

**Comment by aturner-epcc** [comment]
@dme26 Thanks for the link - that is interesting, you learn something every day!

**Comment by jcohen02** [comment]
👍 16th March, 18 UTC works for me too, it's in my diary.

**Comment by ChristinaLK** [comment]
@aturner-epcc my experience is that people equate 'GMT' with British time, which has both a winter/summer time, so isn't consistent when planning meetings, especially around fall/spring. :)

**Comment by sstevens2** [comment]
Ok I'll also send out an email invite.  Please add anything you'd like to be included in the agenda to this thread.  My first thoughts are.

- Clarifying Audience - Creating Learner Profiles for the lesson
- What should be moved to extras
- Things we need to do to polish this lesson

**Comment by ChristinaLK** [comment]
I think we should clarify the role of external computing systems in the goals for this lesson. 

Also discuss how (and what) to sync with the containers lesson developed by Pawsey.

**Comment by aturner-epcc** [comment]
I agree completely with @ChristinaLK on this. One of the strengths of this lesson is that it can all be run on the learner's own laptop/workstation and, by the end of the course, they have a working setup on their system they can use to explore further. It gives a good grounding on container concepts that then make transferring to another container approach on, say, an HPC system easier - however, I do not think we should be attempting to teach that here.

For me, the least satisfactory part of the current Docker lesson is episode 6 Containers in the Cloud as it does not seem to fit as well with the rest of the lesson (in fact, whenever I have used the material I have dropped this episode). I suspect that this is for much the same reason as containers in HPC would not fit here - this is a whole different topic that deserves a lesson of its own.

Maybe the Singularity lesson should be re-titled something like: Singularity: Using Containers on HPC Systems to distinguish its goals better from this lesson.

**Comment by dme26** [comment]
Yes, episode 6 is super-weird, which I feel I can safely say since I'm only being rude to myself! I reckon that there's already plenty of material elsewhere, and that the episode should probably be completely replaced or just removed. If put in extras, it should be reworked to be more carefully thought through than my first attempt...
